packages/components/app/styles/components/form/file-input.scss

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// FORM > FILE-INPUT
+//
+
+@use "../../mixins/button" as *;
+
+.hds-form-file-input {
+  margin: -4px 0 -4px -4px; // offset shadow padding + default 1px spacing
+  padding: 3px 0 3px 3px; // add padding so button shadow is visible
+  color: var(--token-color-foreground-primary);
+
+  &:focus,
+  &:focus-visible {
+    outline: none;
+  }
+}
+
+// https://developer.mozilla.org/en-US/docs/Web/CSS/::file-selector-button
+// https://caniuse.com/?search=file-selector-button
+
+// Button
+.hds-form-file-input::file-selector-button {
+  min-height: 36px;
+  margin-right: 16px;
+  margin-inline-end: 16px;
```

Comment:
I see you have added both. Any specific reason? (if yes, then probably better to add a comment to explain why)

Reply:
I'll test if it's doing anything then delete if not, thanks.

Reply:
I'll test if it's doing anything then delete if not, thanks.